README.md

```diff
@@ -13,6 +13,7 @@
 This SDK currently supports these APIs:
 
 * [Domains API](https://dm.realtimeregister.com/docs/api/domains)
+* [Hosts API](https://dm.realtimeregister.com/docs/api/hosts)
```

Comment:
This doc is empty?

Reply:
Most of them are, they merely point to a page which opens up the sidebar menu, e.g. https://dm.realtimeregister.com/docs/api/customers or https://dm.realtimeregister.com/docs/api/domains are also empty